editor/src/messages/dialog/preferences_dialog/preferences_dialog_message_handler.rs

```diff
@@ -45,6 +45,15 @@ impl PreferencesDialogMessageHandler {
 				})
 				.widget_holder(),
 		];
+		let use_vello = vec![
+			TextLabel::new("Input").min_width(60).italic(true).widget_holder(),
```

Comment:
This probably shouldn't be under input. Also it might be worth writing "experimental" or something in the UI.

Reply:
Yeah, I'd recommend:
"Renderer" and "Vello (experimental)"

Reply:
> This probably shouldn't be under input. Also it might be worth writing "experimental" or something in the UI.

Yeah already have exactly that change locally :D